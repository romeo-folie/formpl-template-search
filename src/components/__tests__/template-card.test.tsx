import renderer from "react-test-renderer";
import TemplateCard from "../template-card/template-card.component";

test("template card matches snapshot", () => {
  const templateProps = {
    name: "Test template",
    category: ["Health"],
    created: "2020-11-04T16:26:44.666569",
    description: "Test description",
    link: "https://formpl.us",
  };
  const templateCard = renderer
    .create(<TemplateCard {...templateProps} />)
    .toJSON();
  expect(templateCard).toMatchSnapshot();
});
