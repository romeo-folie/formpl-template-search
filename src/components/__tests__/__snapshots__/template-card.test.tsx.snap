// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`template card matches snapshot 1`] = `
<div
  className="sc-bdnxRM crfAaz"
>
  <div
    className="sc-gtsrHT KltRp"
  >
    <h3
      className="sc-hKFxyN eKCFkS"
    >
      Test template
    </h3>
    <p
      className="sc-eCApnc cVQGsc"
    >
      Test description
    </p>
  </div>
  <div
    className="sc-dlnjwi dNMoCL"
  >
    <a
      className="sc-jSFjdj hBOqe"
    >
      Use Template
    </a>
  </div>
</div>
`;
